docs/api-reference/geo-layers/tile-layer.md

```diff
@@ -266,8 +267,22 @@ Receives arguments:
 
 Receives arguments:
 
-- `tile` (Object) - the tile that has been cleared from cache.
+- `tile` (Object) - the [tile](#tile) that has been cleared from cache.
+
+## Tile
+
+Class to hold the reading of a single tile
+
+Properties:
+
+- `x` (Number) - x index of the tile
+- `y` (Number) - y index of the tile
+- `z` (Number) - z index of the tile
+- `bbox` (Object) - tile boundaries in WGS84
```

Comment:
This is incorrect. See https://deck.gl/docs/api-reference/geo-layers/tile-layer#gettiledata

Reply:
Fixed